lib/uname.js

```diff
@@ -0,0 +1,43 @@
+var getUname = require('node-uname')
```

Comment:
The hostname and the kernel release can retreived from node's ```os``` module, but as far as i know there's nothing builtin to get the kernel-version...
Some parameters are very similiar but not the same in node's builtin modules like the architecture or the kernel-name.